soccer/gameplay/tests/test_field.py

```diff
@@ -0,0 +1,151 @@
+import unittest
+import main
+import robocup
+import evaluation.field
+import constants
+
+class Moc_Robot:
+	def __init__(self, x, y):
+		self.pos = robocup.Point(x, y)
+		self.visible = True;
+
+	def set_pos(self, x, y):
+		self.pos = robocup.Point(x, y)
+
+class TestField(unittest.TestCase):
+	def __init__(self, *args, **kwargs):
+		super(TestField, self).__init__(*args, **kwargs)
+
+	def setUp(self):
+		self.robots = [Moc_Robot(0, 0), Moc_Robot(0, 0), Moc_Robot(0, 0), 
+				Moc_Robot(0, 0), Moc_Robot(0, 0), Moc_Robot(0, 0)]
+		main.set_their_robots(self.robots)
+
+	# Set 6 robots position to a single point
+	#
+	# @param x final x coordinate of robots
+	# @param y final y coordinate of robots
+	#
+	def set_robot_pos(self, x, y):
+		for robot in self.robots:
+			robot.set_pos(x, y)
+		main.set_their_robots(self.robots)
+
+	# Positions 6 robots around a point but not on it
+	#
+	# @param x: x coordinate of point
+	# @param y: y coordinate of point
+	# 
+	def set_robots_around_pos(self, x, y, rad):
+		self.robots[0].set_pos(x - constants.Robot.Radius * rad, y - constants.Robot.Radius * rad)
+		self.robots[1].set_pos(x - constants.Robot.Radius * rad, y + constants.Robot.Radius * rad)
+		self.robots[2].set_pos(x, y + constants.Robot.Radius * rad)
+		self.robots[3].set_pos(x + constants.Robot.Radius * rad, y - constants.Robot.Radius * rad)
+		self.robots[4].set_pos(x + constants.Robot.Radius * rad, y)
+		self.robots[5].set_pos(x, y - constants.Robot.Radius * rad)
```

Comment:
This might be cleaner if you create a variable with the value `constants.Robot.Radius * rad`